orangecontrib/text/widgets/owcorpusviewer.py

```diff
@@ -124,7 +133,9 @@ def set_data(self, data=None):
             if not isinstance(data, Corpus):
                 self.corpus = Corpus.from_table(data.domain, data)
             self.load_features()
+            self.filter_input.setText(self.regexp_filter)
```

Comment:
Instead of doing this, can you rather refactor `self.filter_input` to automatically sync the value with `self.regexp_filter`?